pubspec.yaml

```diff
@@ -9,6 +9,9 @@ homepage: https://github.com/dart-lang/typed_data
 environment:
   sdk: '>=2.4.0 <3.0.0'
 
+dependencies:
+  collection: ^1.1.0
+
 dev_dependencies:
   pedantic: ^1.0.0
```

Comment:
Could also just not depend on pedantic, or lock it to pedantic: 1.7.0 are thereabout.
The newest versions of pedantic have jumped the shark, and I do not want to support the lints they have chose to enable.

Reply:
Done.